src/pages/Earn/index.tsx

```diff
@@ -48,16 +81,27 @@ export default function Earn() {
   const { t } = useTranslation()
   const [filteringToken, setFilteringToken] = useTokenFilter()
   const farmSummaries = useFarmRegistry()
+  const [depositSort, setDepositSort] = useState<boolean>(false)
```

Comment:
Use an `enum` rather than a `boolean` for sorts since they are mutually exclusive

enum FarmSort {
  DEPOSIT,
  YIELD,
}